src-vue/src/plugins/store.ts

```diff
@@ -57,72 +57,82 @@ export const store = createStore({
             state.current_tab = newTab;
         },
         async launchGame(state: any) {
-            // TODO update installation if release track was switched
-
-            // Install northstar if it wasn't detected.
-            if (state.northstar_state === NorthstarState.INSTALL) {
-                let install_northstar_result = invoke("install_northstar_caller", { gamePath: state.game_path, northstarPackageName: ReleaseCanal.RELEASE });
-                state.northstar_state = NorthstarState.INSTALLING;
-
-                await install_northstar_result.then((message) => {
-                    console.log(message);
-                })
-                    .catch((error) => {
-                        console.error(error);
-                        alert(error);
-                    });
-
-                _get_northstar_version_number(state);
-                return;
-            }
+            switch (state.northstar_state) {
+                // TODO update installation if release track was switched
+
+                // Install northstar if it wasn't detected.
+                case NorthstarState.INSTALL:
+                    {
```

Comment:
You don't need to use brackets in switch-case :)

Reply:
I'm well aware but you have scoped defines with `let` that will error out otherwise without scoped blocks :P